src/wallet/sqlite.cpp

```diff
@@ -124,3 +125,8 @@ bool SQLiteBatch::TxnAbort()
 {
     return false;
 }
+
+std::string SQLiteDatabaseVersion()
```

Comment:
In commit "Implement SQLiteDatabaseVersion" (b2f2e7171861fe2998f2c27af44b65b0e1f558aa)

Minor: SQLiteLibraryVersion might be a more descriptive name

Reply:
Probably, but I'm also following the convention set by `BerkeleyDatabaseVersion`.